EngineLayer/EngineLayer.csproj

```diff
@@ -64,9 +64,6 @@
     <None Update="ProteolyticDigestion\proteases.tsv">
       <CopyToOutputDirectory>Always</CopyToOutputDirectory>
     </None>
-    <None Update="settings.toml">
```

Comment:
why delete the settings.toml

Reply:
it's generated automatically if it doesn't exist now